includes/html/pages/addhost.inc.php

```diff
@@ -241,6 +241,12 @@
               <select name="authalgo" id="authalgo" class="form-control input-sm">
                 <option value="MD5" selected>MD5</option>
                 <option value="SHA">SHA</option>
+                <?php if (snmpv3_sha2_capable()) {?>
```

Comment:
I'm no fan of hiding a feature because a "hidden" dependency requirement is not met.  
Maybe set them as disabled and  print info text if the requirement is not met?